editions/tw5.com/tiddlers/filters/syntax/then Filter Run Prefix.tid

```diff
@@ -0,0 +1,113 @@
+created: 20210618133745003
+from-version: 5.3.0
+modified: 20230506172920710
+rp-input: <<.olink all>> tiddler titles, unless otherwise determined by the first [[filter operator|Filter Operators]]
+rp-output: the output of this filter run replaces the output of previous runs unless it is an empty list (see below)
+rp-purpose: replace any input to this filter run with its output
```

Comment:
Hi @saqimtiaz, I think I need some help here. I find it exceedingly hard to wordsmith a one-liner that encompasses the FRP behavior in all edge cases. `rp-purpose` should in my opinion by as concise as possible, similar to the filter operators, since otherwise the utility of summary tiddlers such as "Filter operators" is greatly diminished.
If you have something in mind, I'll gladly use it.

Reply:
Perhaps something along these lines would be helpful:
`evaluate the filter run only when there is any input to it, replacing the input with its output.`

Reply:
I'll adapt that. 👍